lib/ansible/modules/cloud/azure/azure_rm_appgateway.py

```diff
@@ -232,38 +234,42 @@
         suboptions:
             name:
                 description:
-                    - Name
+                    - Name.
             protocol:
                 description:
-                    - Protocol
+                    - Protocol.
                 choices:
                     - 'http'
                     - 'https'
             host:
                 description:
-                    - Host
+                    - Host.
             path:
                 description:
-                    - Path
+                    - Path.
+                    - Valid path starts from '/'. Probe is sentto <Protocol>://<host>:<port><path>.
             timeout:
                 description:
-                    - Timeout
+                    - Timeout.
+                    - Acceptable values are from 1 second to 86400 seconds.
             interval:
                 description:
-                    - Interval
+                    - Interval.
+                    - Acceptable values are from 1 second to 86400 seconds.
             unhealthy_threshold:
                 description:
-                    - Unhealthy Threshold
+                    - Unhealthy Threshold.
+                    - Acceptable values are from 1 second to 20.
     backend_http_settings_collection:
         description:
             - Backend http settings of the application gateway resource.
         suboptions:
             probe:
                 description:
-                    - Probe
+                    - Probe.
```

Comment:
While you're updating these lines, can you add a little more detail? Probe for what?

Reply:
Thanks @Fred-sun for the details!